src/nav.js

```diff
@@ -87,6 +91,15 @@ export default function PersistentDrawerLeft() {
     setOpen(false);
   };
 
+  const [user, setUser] = useState(null);
+
+  async function loginPress(){
+    const user = await Authentication.logIn();
+    if (user === null) { /* error */ }
```

Comment:
Can you display to the user that the log in failed? Using an alert would be a minimum. A custom UI component or text indicating so would be ideal.